mkstage4.sh

```diff
@@ -10,10 +10,20 @@ fi
 #set flag variables to null
 EXCLUDE_BOOT=0
 EXCLUDE_CONNMAN=0
+EXCLUDE_LOST=1
 QUIET=0
+USAGE="usage:\n\
+  `basename $0` [-q -c -b] [-s || -t <target-mountpoint>] <archive-filename> [custom-tar-options]\n\
+  -q: activates quiet mode (no confirmation).\n\
+  -c: excludes connman network lists.\n\
+  -b: excludes boot directory.\n\
+  -l: includes lost+found directory.\n\
```

Comment:
Since for all the other `-*` options we exclude a directory, it would be confusing to have `-l` include one. I realize that more commonly you would want to exclude it - but I think it would eb bette to prioritize argument consistency.